src/Api.js

```diff
@@ -1,11 +1,25 @@
 export async function getVans() {
-  const res = await fetch("api/vans");
+  const res = await fetch("/api/vans");
+  if (!res.ok) {
+    throw {
+      message: "Failed to fetch vans",
+      statusText: res.statusText,
+      status: res.status,
+    };
```

Comment:
Since this portion of code here is used more than once, you can put it in a function and then be calling that function whenever you need it to avoid repeating yourself.